Bottom: 2cdb185935e1aeec52530bf3e4d27b5465ab8647
Top:    2fb0c1e07a075571b1ddd2dd06ccf2f8e1d3d2c2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 23:24:40 +0000

Refresh of temp-0

---

diff --git a/src/polygon.c b/src/polygon.c
index f488d79..e9cac40 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -108,7 +108,7 @@ dicer output is used for HIDs which cannot render things with holes
 #include <sys/time.h>
 #include <sys/resource.h>
 
-#define DEBUG_CIRCSEGS
+#undef DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
